winpr/include/winpr/wtypes.h

```diff
@@ -49,17 +57,39 @@
 #define NEAR
 #endif
 
+#if defined(HAVE_STDINT_H)
```

Comment:
@akallabeth ok so I don't understand as what I'm reading is that when stdint.h is detected you (re)define int8_t as char and so on. Am I missing something ?